docs/src/operations/health.md

```diff
@@ -0,0 +1,29 @@
+# Health Status
+
+Zeebe broker exposes two http endpoints to query its health status. 
+* Ready check
+* Health check
+ 
+## Ready Check
+Ready check endpoint is exposed via `http://{zeebe-broker}:{zeebe.broker.network.monitoringApi.port}/ready`.
+This endpoint return an empty 204 response. If it is not ready, it will return a 503 error.
+
+A broker is ready when it has installed all necessary services to start processing in all partitions. 
+Once it is ready it is always ready.
+
+A ready check is useful, for example, to use as a `readinessProbe` in a kubernetes configuration to control when a pod can be restarted for rolling upgrade. 
+Depending on the cluster configuration, restarting one pod before the previous one is ready might make the system unavailable because the quorum of replicas is not available.
+By configuring a `readinessProbe` that uses the ready check endpoint we can inform Kubernetes when it is safe to proceed with rolling update.
```

Comment:
We accept requests already before right? But not for all partitions, maybe we add that the broker will after he is ready he can work on all his partitions, then probably we need to add that `ready != leader available`

Reply:
:+1: I have added that above. Please take a look.